### PR TITLE
[13.0][MIG] project_timeline_hr_timesheet: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
**Depends on** https://github.com/OCA/project/pull/656

timeline_hr_timesheet

cc @tecnativa TT19799

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]
Code review..

File: project_timeline_hr_timesheet/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Dennis Sluijk <d.sluijk@onestein.nl>
+* Nikul Chaudhary <nikulchaudhary2112@gmail.com>
+* Alexandre Díaz <alexandre.diaz@tecnativa.com>
```
Comment (manuelcalerosolis) [inline project_timeline_hr_timesheet/readme/CONTRIBUTORS.rst]: * `Tecnativa <https://www.tecnativa.com>`__:
  * Alexandre Díaz

**Review by manuelcalerosolis — APPROVED** [review]
Code and functional review.

**Comment by Tardo** [comment]
Changes done!

**Comment by pedrobaeza** [comment]
Please rebase and check its correct behavior.

**Comment by Tardo** [comment]
Rebased and cleaned 'bot' commits

**Pull request merged**